package/CHANGELOG

```diff
@@ -20,6 +20,7 @@ The rules for this file:
   * 1.0.1
 
 Fixes
+  * Disable NSGrid in favour fo PKDTree (Issue #2930)
```

Comment:
I think we need to state that the nsgrid implementation could lead to wrong results (similar to what you have in you versionchanged messages).

Reply:
Also:
```python
"fo"[::-1]
```

Reply:
I would claim that this is caused by writing docs at 2 am, but I'm pretty sure I make just as many typos during the day 🙄 

Should be fixed in ffc0a72 (along with more typos 😅 )